File: TestFloatingActionButton/app/src/main/java/com/pix/testfloatingactionbutton/MainActivity.java
package com.pix.testfloatingactionbutton;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void normal1Click(View v) {
        startActivity(new Intent(this,NormalOneActivity.class));
    }

    public void customClick(View v) {
        startActivity(new Intent(this,CustomActivity.class));
    }
}

File: TestDrawerLayout/app/src/main/java/com/pix/testdrawerlayout/MainActivity.kt
package com.pix.testdrawerlayout

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), View.OnClickListener {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        btn_simple.setOnClickListener(this)
        btn_toolbar.setOnClickListener(this)
    }

    override fun onClick(v: View?) {
        when(v) {
            btn_simple -> {
                startActivity(Intent(this,SimpleDrawerLayoutActivity::class.java))
            }
            btn_toolbar -> {
                startActivity(Intent(this,ToolbarDrawerActivity::class.java))
            }
        }
    }
}
